playbooks/luna.yml

```diff
@@ -21,6 +21,7 @@
       - "--enable-foreman-cli-remote-execution"
       - "--enable-foreman-cli-tasks"
       - "--enable-foreman-cli-templates"
+      - "--enable-ostree"
     foreman_installer_additional_packages:
       - tfm-rubygem-foreman_virt_who_configure
```

Comment:
https://github.com/theforeman/foreman-installer/pull/371 should expose these too

Reply:
I agree, can you add it there please? I think I could merge that PR in return :-)